dataland-frontend/src/components/resources/datasetOverview/DatasetOverviewTable.vue

```diff
@@ -0,0 +1,138 @@
+        <template #body="{ data }">
+          <span>{{ convertDate(data.uploadTimeInMs) }}</span>
+        </template>
+      </Column>
+      <Column field="companyName" header="" class="w-2 d-bg-white d-datatable-column-right">
+        <template #header>
+          <span class="w-12 p-input-icon-left">
+            <i class="pi pi-search pl-3 pr-3" aria-hidden="true" style="color: #958d7c" />
+            <InputText v-model="searchBarInput" placeholder="Search table" class="w-12 pl-6" />
+          </span>
+        </template>
+        <template #body="{ data }">
+          <router-link :to="getTableRowLinkTarget(data)" class="text-primary no-underline font-bold">
+            <div class="text-right">
+              <span>VIEW</span>
+              <span class="ml-3">></span>
+            </div>
+          </router-link>
+        </template>
+      </Column>
+    </DataTable>
+  </div>
+</template>
+
+<style>
+#dataset-overview-table tr:hover {
+  cursor: pointer;
+}
+
+#dataset-overview-table th {
+  padding-top: 0.8rem;
+  padding-bottom: 0.8rem;
+}
+</style>
+
+<script lang="ts">
+import { defineComponent, inject } from "vue";
+import DataTable from "primevue/datatable";
+import Column from "primevue/column";
+import { humanizeString } from "@/utils/StringHumanizer";
+import { DatasetTableInfo, getMyDatasetTableInfos } from "@/components/resources/datasetOverview/DatasetTableInfo";
+import InputText from "primevue/inputtext";
+import { convertUnixTimeInMsToDateString } from "@/utils/DateFormatUtils";
+import { DataTypeEnum } from "@clients/backend";
+import Keycloak from "keycloak-js";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+
+export default defineComponent({
+  name: "DatasetOverviewTable",
+  components: {
+    DataTable,
+    Column,
+    InputText,
+  },
+  data() {
+    return {
+      searchBarInput: "",
+      displayedDatasetTableInfos: [] as DatasetTableInfo[],
+      humanizeString: humanizeString,
+      convertDate: convertUnixTimeInMsToDateString,
+    };
+  },
+  setup() {
+    return {
+      getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
+    };
+  },
+  props: {
+    datasetTableInfos: {
+      type: Array,
+      default: [] as DatasetTableInfo[],
+    },
+  },
+  watch: {
+    searchBarInput() {
+      void this.applySearchFilter();
+    },
+    datasetTableInfos() {
+      this.displayedDatasetTableInfos = this.datasetTableInfos as DatasetTableInfo[];
+    },
+  },
+  methods: {
+    /**
+     * Computes the path a row click should lead to depending on the selected dataset and its status
+     *
+     * @param datasetTableInfo relevant dataset information
+     * @returns the path depending on the status of the data set
+     */
+    getTableRowLinkTarget(datasetTableInfo: DatasetTableInfo): string {
+      const dataTypesWithSingleView = [DataTypeEnum.Lksg] as DataTypeEnum[];
```

Comment:
Does this mean LKSG data is used as a default data to be shown?

Reply:
This array defines the frameworks for which all datasets are displayed at the same time. This is currently only LkSG. The taxonomy pages only display a single dataset thats why I used a link with the dataId as parameter.